src/drive/web/modules/search/components/SuggestionItemTextSecondary.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+
+import AppLinker, {
+  generateWebLink,
+  generateUniversalLink
+} from 'cozy-ui/transpiled/react/AppLinker'
+import { useClient } from 'cozy-client'
+import { isFlagshipApp } from 'cozy-device-helper'
+
+import styles from 'drive/web/modules/search/components/styles.styl'
+import SuggestionItemTextHighlighted from 'drive/web/modules/search/components/SuggestionItemTextHighlighted'
+
+const SuggestionItemTextSecondary = ({
+  text,
+  query,
+  url,
+  onOpened,
+  isMobile
+}) => {
+  const client = useClient()
+
+  if (isMobile) {
+    return <SuggestionItemTextHighlighted text={text} query={query} />
+  }
+
+  const app = {
+    slug: 'drive'
+  }
+
+  const { subdomain: subDomainType } = client.getInstanceOptions()
```

Comment:
@Crash-- is it a correct way to access subdomain or I should prefer the code below ?
```
const { capabilities } = useCapabilities(client)
  const isFlatDomain = useMemo(() => {
    return get(capabilities, 'flat_subdomains')
  }, [capabilities])
```